Servers/how-to-update-softnas.md

```diff
@@ -0,0 +1,188 @@
+{{{
+  "title": "SoftNAS: How to update",
+  "date": "11-11-2015",
+  "author": "Maxim Volkov",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": true
```

Comment:
As simply a way to update a 3rd party product I don't think this should be a sticky/must read kb.